app/src/main/java/org/covidwatch/android/di/AppModule.kt

```diff
@@ -28,6 +29,13 @@ val appModule = module {
         context.getSharedPreferences("org.covidwatch.android.PREFERENCE_FILE_KEY", Context.MODE_PRIVATE)
     }
 
+    viewModel {
+        MainViewModel(
+            testRepository = get(),
```

Comment:
Can we rename `testRepository` at some point?

Reply:
yeah for sure! you have something in your mind?

Reply:
Honestly I'd prefer `Tested`, `IsTested`, or `IsTestedPositiver`, just because `Test` can be ambiguous. A little weird still but more specific

Reply:
Renamed to `Tested`